lib/elixir_analyzer/cli.ex

```diff
@@ -6,22 +6,19 @@ defmodule ElixirAnalyzer.CLI do
   @usage """
   Usage:
 
-    $ elixir_analyzer <exercise-name> <input path> <output path>  [options]
+    $ elixir_analyzer <exercise-name> <input path> <output path> [options]
 
   You may also pass the following options:
-    --skip-analysis                       flag skips running the static analysis
-    --output-file <filename>
-
-  You may also test only individual files :
-    (assuming analyzer tests are compiled for the named module)
-
-    $ exercism_analyzer --analyze-file <full-path-to-.ex>:<module-name>
```

Comment:
This was an option in the CLI, but could not actually be passed to the analyzer, so I got rid of it